src/api/auth/auth.controller.ts

```diff
@@ -4,18 +4,27 @@
 import { AuthService } from './auth.service';
 import { LoginDto } from './dto/login.dto';
 
+
 @ApiTags('Auth')
 @Controller('auth')
 export class AuthController {
   constructor(private readonly authService: AuthService) {}
 
   @Post('sign-up')
-  async register(@Body() register: CreateUserDto) {
+  async register(@Body() register: CreateUserDto, hre: any) {
+    
+    let var1;
+    if (register.username === '') {
+      var1 = 'Username is required';
+    } else {
+      var1 = ''
+    }
+    console.log(var1);
```

Comment:
🚫 **[eslint]** <[unicorn/prevent-abbreviations](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v56.0.0/docs/rules/prevent-abbreviations.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The variable `var1` should be named `variable1`. A more descriptive name will do too.
```suggestion
    let variable1;
    if (register.username === '') {
      variable1 = 'Username is required';
    } else {
      variable1 = ''
    }
    console.log(variable1);
```

<!-- __reviewdog__:Cg9iODA3OTNiNWI4ODQ4YWYSBmVzbGludA== -->